contracts/mainnet/connectors/pendle/interfaces.sol

```diff
@@ -0,0 +1,104 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+interface IPendle {
+    struct ApproxParams {
+        uint256 guessMin;
+        uint256 guessMax;
+        uint256 guessOffchain; // pass 0 in to skip this variable
+        uint256 maxIteration; // every iteration, the diff between guessMin and guessMax will be divided by 2
+        uint256 eps; // the max eps between the returned result & the correct result, base 1e18. Normally this number will be set
+        // to 1e15 (1e18/1000 = 0.1%)
+    }
+
+    struct TokenInput {
+        // TOKEN DATA
+        address tokenIn;
+        uint256 netTokenIn;
+        address tokenMintSy;
+        // AGGREGATOR DATA
+        address pendleSwap;
```

Comment:
@shriyatyagii Can't we hardcode this? Or does this change frequently.

Reply:
`netTokenIn` would chage, rest we can hardcode